update-timescaledb/update-tsdb-2.md

```diff
@@ -0,0 +1,187 @@
+# Updating TimescaleDB to 2.0 [](update)
+
+Use these instructions to update TimescaleDB 1.x to TimescaleDB 2.0
+
+
+>:WARNING:These instructions are only for upgrading TimescaleDB 1.x to TimescaleDB 2.0
+> If you need to upgrade your existing TimescaleDB 1.x to a newer version in the 1.x
+> release line (eg. 1.7.2 to 1.7.4), please see Update [TimescaleDB 1.x][update-tsdb-1]
+
+### TimescaleDB Release Compatibility
+
+TimescaleDB 2.0 currently supports the following PostgreSQL releases. If you are not currently running 
+a compatible release, please upgrade before updating TimescaleDB.
+
+ TimescaleDB Release |   Supported PostgreSQL Release
+ --------------------|-------------------------------
+ 2.0-RC1+            | 11, 12
+
+>:TIP:If you need to upgrade PostgreSQL first, please see [our documentation][upgrade-pg].
+
+### Notice of breaking changes from TimescaleDB 1.3+
+TimescaleDB 2.0 supports **in-place updates** just like previous releases. During the update, scripts will automatically configure
+updated features to work as expected with TimescaleDB 2.0.
+
+Because this is our first major version release in two years, however, we’re providing additional guidance to help you ensure the update completes successfully and everything is configured as expected (and optimized for your workload). In particular, settings related to [Continuous Aggregates][caggs], [compression][compression], and [data retention][retention] have been modified to provide greater configuration transparency and flexibility, therefore we highly recommend verifying that these settings were migrated correctly.
```

Comment:
Small nit - generally recommend using hard line breaks, so that later diffs can be more easily compared via git at the line (vs. paragraph) level.